app/components/Views/confirmations/components/ApproveTransactionReview/index.js

```diff
@@ -611,7 +615,9 @@ class ApproveTransactionReview extends PureComponent {
       tokenSpendValue,
       originalApproveAmount,
     } = this.state;
-    Analytics.trackEvent(MetaMetricsEvents.TRANSACTIONS_EDIT_TRANSACTION);
+    this.props.metrics.trackEvent(
```

Comment:
destructure metrics on L606
```suggestion
    metrics.trackEvent(
```